Makefile

```diff
@@ -8,8 +8,8 @@ test-all-features:
 # would skip it.
 cov:
 	cargo clean
-	cargo test --all --features=asm -- --nocapture
-	for file in `find target/debug/ -maxdepth 1 -executable -type f`; do mkdir -p "target/cov/$$(basename $$file)"; kcov --exclude-pattern=/.cargo,/usr/lib,tests "target/cov/$$(basename $$file)" "$$file"; done
+	cargo test --no-run --all --features=asm -- --nocapture
```

Comment:
If we just want to build tests only, we can use `cargo build --tests`

Reply:
I think they are the same.

Reply:
I vote for `cargo build --tests`, yes the functionalities are the same, but `cargo build --tests` states what we are achieving here better.